### PR TITLE
Help Center: Update useLaunchpad hook

### PR DESCRIPTION
**Pull request by edanzer**
### Proposed Changes

This PR simply updates the Help Center > Launchpad icon component to use the new useLaunchpad hook from data stores. That hook, in turn, leverages our new Jetpack backend endpoint. Behavior should be the same as before, although given the improved backend code, the progress indicator on the Launchpad icons should now generally be correct. 

### Testing Instructions

You no longer need to sync a Jetpack branch to your sandbox.

1. Checkout this branch and run yarn and yarn start. This should build the help center package fresh for testing in addition to starting your local dev environment. Note that the relevant changes here are in a package. I sometimes needed to run `yarn build-packages` to force the help center package to rebuild with latest changes. 
3. You will need to create or test with a Launchpad enabled site. 
4. Go to http://calypso.localhost:3000/posts/YOURSITESLUG
4. TEST: Click on the help / question mark icon in the upper right icon. You should see the Launchpad icon. Historically, this has always rendered with the progress bar at 25% complete. It should now be dynamically based on the total and completed site steps. Once you've seen this once, consider navigating back to Launchpad, completing additional task(s), and then checking /posts/ again to see if the progress on the Help Center Launchpad icon has updated.

### CODE REVIEW

**Comment by jeyip** [comment]
## Testing
### Requirements
- [x] TEST: Click on the help / question mark icon in the upper right icon. You should see the Launchpad icon. It should now be dynamically based on the total and completed site steps.
- [x] Once you've seen this once, consider navigating back to Launchpad, completing additional task(s), and then checking /posts/ again to see if the progress on the Help Center Launchpad icon has updated.

### Browsers
- [x] Chrome

**Comment by jeyip** [comment]
This works as expected on simple sites, although won't work properly on atomic sites until the latest `jetpack-mu-wpcom` updates are released through `wpcomsh`. Let's wait until those changes are live before merging this

**Review by jeyip — APPROVED** [review]

**Comment by agrullon95** [comment]
Simple site test:
Verified the help center launchpad circular progress bar is displaying correctly and is updating when I complete a task on the launchpad screen.

Waiting to approve when latest jetpack-mu-wpcom are deployed to atomic via wpcomsh.

**Pull request merged**
